.pre-commit-config.yaml

```diff
@@ -48,7 +48,7 @@ repos:
       - id: rstcheck
         additional_dependencies:
           - tomli
-        files: ^doc/.*\.(rst|inc)$
+        files: ^mne/|^doc/
```

Comment:
This was actually not what I was suggesting to change, this is the `rstcheck` run. I was thinking one of the `ruff` runs, probably the examples/tutorials would be best:

https://github.com/mne-tools/mne-python/blob/2f0d2f3ee2422dfb721d1bcc7138ff1a6fccd662/.pre-commit-config.yaml#L22

Reply:
Sorry about that. This Ruff configuration is still abstract to me since there are `ruff-format`, `ruff format mne`, `ruff format tutorials and examples`, `ruff lint tutorials examples` etc. I just changed line 22 `ruff lint tutorials and examples` to `^mne/|^doc/` .

Reply:
Hey @larsoner FYI I think me amending line 22 as above caused a big behaviour change. See [5619342](https://github.com/mne-tools/mne-python/pull/12237/commits/5619342b6915bf6ecfa01d9ffac989e5b6b869b7).

Reply:
When you said this I expected hundreds of lines changed but it was only in the tens, so I think it's okay just to clean up / unify here. I've pushed some commits to do it

Reply:
Thank you! @larsoner 🙏  🙏